access/ms-teams/Makefile

```diff
@@ -0,0 +1,77 @@
+VERSION=9.3.9
+GO_VERSION=1.17.11
```

Comment:
Should we use the latest release now that Teleport 10 is out? It also uses Go 1.18.

Reply:
With pleasure :-)